sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -1616,11 +1616,10 @@ class StatisticsSuite extends StatisticsCollectionTestBase with TestHiveSingleto
         Seq(tbl, ext_tbl).foreach { tblName =>
           sql(s"INSERT INTO $tblName VALUES (1, 'a', '2019-12-13')")
 
-          val expectedSize = 690
```

Comment:
Can we compare with the size as unexpected before insertion instead?

Reply:
when `spark.sql.statistics.size.autoUpdate.enabled` is `false` (it's the default value), table stats is None until executing `ANALYZE TABLE ...`

I update the test to reflect that.